bukkit/src/main/resources/config.yml

```diff
@@ -456,4 +460,4 @@ default-assignments:
 #      - group.default
 #    give:
 #      - group.member
-#    set-primary-group: member
\ No newline at end of file
+#    set-primary-group: member
```

Comment:
What's happened here? :o

Reply:
I can't seem to remove the "new line at the end of the file." Odd because it's not something I was even modifying, perhaps a side effect of using github online commit function ?

Reply:
Most likely - thanks for trying :)